src/api.js

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://news-api-5gcc.onrender.com/api",
+});
+
+
+const getArticles = () => {
+    return api.get("/articles").then(({ data }) => {
+      return data.articles;
+    });
+  };
+
+
+
+export {getArticles}
```

Comment:
Very nice use of `axios.create` this will make your API call functions very well organised